compiler/rustc_mir/src/borrow_check/diagnostics/conflict_errors.rs

```diff
@@ -1268,6 +1269,27 @@ impl<'cx, 'tcx> MirBorrowckCtxt<'cx, 'tcx> {
 
         if return_span != borrow_span {
             err.span_label(borrow_span, note);
+
+            let tcx = self.infcx.tcx;
+            let ty_params = ty::List::empty();
+
+            let return_ty = self.regioncx.universal_regions().unnormalized_output_ty;
+            let return_ty = tcx.erase_regions(return_ty);
+
+            // to avoid panics
+            if !return_ty.has_infer_types() {
+                if let Some(iter_trait) = tcx.get_diagnostic_item(sym::Iterator) {
+                    if tcx.type_implements_trait((iter_trait, return_ty, ty_params, self.param_env))
+                    {
+                        err.span_suggestion_short(
+                            borrow_span,
+                            "Use `.collect()` to allocate the iterator",
```

Comment:
```suggestion
                            "use `.collect()` to allocate the iterator",
```
I believe that we prefer all our normal diagnostic messaging to be lowercase.

Reply:
Thank you for catching that